pkg/resources/view.go

```diff
@@ -684,6 +746,53 @@ func handleDataMetricFunctions(ctx context.Context, client *sdk.Client, id sdk.S
 	})
 }
 
+func handleColumns(d *schema.ResourceData, columns []sdk.ViewDetails, policyRefs []sdk.PolicyReference) error {
```

Comment:
so much logic here, could we add a unit test for it? I know that ResouceData is limited but for the test we could provide something else implementing d.Set() and check the interations